src/coreclr/jit/lsra.h

```diff
@@ -2259,6 +2279,14 @@ class RefPosition
     // would be 0..MAX_RET_REG_COUNT-1.
     unsigned char multiRegIdx : 2;
 
+#ifdef TARGET_ARM64
+    // If this refposition needs consecutive register assignment
+    bool needsConsecutive;
+
+    // How many consecutive registers does this and subsequent refPositions need
+    unsigned char regCount : 3;
+#endif // TARGET_ARM64
```

Comment:
What is the impact on the size of `RefPosition` with these additional fields?

Reply:
None. Before and after they remain 48 bytes.

  | Name | Value 
-- | -- | ---|
1.  | sizeof(Referenceable*) | 8 
2.  | sizeof(RefPosition*) | 8 
3.  | sizeof(GenTree*) | 8 
4.  | sizeof(bbNum) | 4 
5.  | sizeof(LsraLocation) | 4 
6.  | sizeof(regMaskTP) | 8 
7.  | sizeof(RefType) | 1 
8.  | sizeof(bit-fields) | 20 bits